spec/controllers/submissions_controller_spec.rb

```diff
@@ -19,4 +19,34 @@
         .to change { Submission.where(submission_attrs).exists? }.to(true)
     end
   end
+
+  describe '#approve' do
+    let!(:pending_submission) { Submission.create!(body: 'Pending submission') }
+    let(:params) { { submission_id: pending_submission.id } }
+
+    context 'with admin authentication' do
+      let(:admin) do
+        Admin.create!(email: 'admin@example.com', password: 'Password1')
+      end
+
+      before do
+        http_login(username: admin.email, password: admin.password)
+      end
+
+      it 'approves a submission' do
+        expect { patch :approve, params: params }.to(
```

Comment:
@matt-wratt I also don't like the Rubocop rule that requires `change` etc to be parenthesized.

Reply:
I'm open to changing that in specs only as well.

Reply:
Ditto here. It might be idiomatic Ruby, but it's not idiomatic RSpec.

Reply:
I'm all for RSpec specific Rubocop rules too - here is a (now really old) Rubocop ruleset I wrote with a lot of spec specific exclusions (and comments explaining them): https://gist.github.com/mhaylock/21973608ae5a6967d79ea7b0a9e06b19

Reply:
@mhaylock Thanks, that will be good to refer to as needed. For now have just excluded the problematic rules.